### DISCUSSION TITLE
How do I {...bind()} if I don't have access to the DOM in a react-three-fiber component? Can I provide an element?

### DISCUSSION THREAD

**Discussion by yuchant** [pmndrs/use-gesture#313] [Q&A] [answered]
Can I provide a DOM element to `bind()` somehow?  

I suppose I pass a ref down from the root, and manually add/remove each event handler returned by bind?

Thank you.

**Comment by dbismut** [comment, marked as answer]
Not sure what you mean: you can pass `{... bind()}` to a r3f `<mesh {...bind()} />`, no problem.
The lib supports passing a ref but it needs to be an HTMLElement:

```js
useDrag(handler, { target: yourRef })
```

Maybe if you describe your use case in more details it might be clearer.

**Reply by yuchant** [reply]
I finally found `config.domTarget` seems to be what I'm looking for. https://use-gesture.netlify.app/docs/options/#domtarget

So can also use `target` for a `ref`! Thank you. Thank you!

Working with the scroll wheel and inertia has been challenging.

**Reply by dbismut** [reply]
You can use `@use-gesture/react` in v10 beta which renames `domTarget` into `target`.
